pkg/controller/datavolume-controller.go

```diff
@@ -632,6 +635,8 @@ func newPersistentVolumeClaim(dataVolume *cdiv1.DataVolume) (*corev1.PersistentV
 		}
 	} else if dataVolume.Spec.Source.Upload != nil {
 		annotations[AnnUploadRequest] = ""
+	} else if dataVolume.Spec.Source.Blank != nil {
+		annotations[AnnBlankImage] = ""
```

Comment:
Same here should be something like 

  annotations[AnnSource] = Blank

Where Blank is "Blank"